ceres build failed

OS: Win10 
MSVS2017
I try to install ceres library but it fails. See the followed log:

> PS C:\Users\DevUser> .$env:vcpkg install ceres:x64-windows
> The following packages will be built and installed:
>     ceres:x64-windows
> Building package ceres:x64-windows...
> -- CURRENT_INSTALLED_DIR=L:/vcpkg/installed/x64-windows
> -- DOWNLOADS=L:/vcpkg/downloads
> -- CURRENT_PACKAGES_DIR=L:/vcpkg/packages/ceres_x64-windows
> -- CURRENT_BUILDTREES_DIR=L:/vcpkg/buildtrees/ceres
> -- CURRENT_PORT_DIR=L:/vcpkg/ports/ceres/.
> -- Using cached L:/vcpkg/downloads/ceres-solver-ceres-solver-1.13.0.tar.gz
> -- Testing integrity of cached file...
> -- Testing integrity of cached file... OK
> -- Extracting source L:/vcpkg/downloads/ceres-solver-ceres-solver-1.13.0.tar.gz
> -- Extracting done
> -- Applying patch L:/vcpkg/ports/ceres/fix-find-packages.patch
> -- Applying patch failed. This is expected if this patch was previously applied.
> -- Applying patch L:/vcpkg/ports/ceres/fix-find-packages.patch done
> -- Configuring x64-windows-rel
> CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:43 (message):
>     Command failed: L:/vcpkg/downloads/cmake-3.9.1-win32-x86/bin/cmake.exe;L:/vcpkg/buildtrees/ceres/src/ceres-solver-1.
> 13.0;-DEXPORT_BUILD_DIR=ON;-DBUILD_EXAMPLES=OFF;-DBUILD_TESTING=OFF;-DCXSPARSE=ON;-DEIGENSPARSE=ON;-DSUITESPARSE=ON;-DGF
> LAGS_PREFER_EXPORTED_GFLAGS_CMAKE_CONFIGURATION=OFF;-DGLOG_PREFER_EXPORTED_GLOG_CMAKE_CONFIGURATION=OFF;-DBUILD_SHARED_L
> IBS=ON;-DVCPKG_TARGET_TRIPLET=x64-windows;-DCMAKE_CXX_FLAGS= /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP ;-DCMAKE_C_FLAG
> S= /DWIN32 /D_WINDOWS /W3 /utf-8 /MP ;-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON;-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON;
> -DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON;-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE;-DCMAKE_VERBOSE_MAKEFIL
> E=ON;-DVCPKG_APPLOCAL_DEPS=OFF;-DCMAKE_TOOLCHAIN_FILE=L:/vcpkg/scripts/buildsystems/vcpkg.cmake;-DCMAKE_ERROR_ON_ABSOLUT
> E_INSTALL_DESTINATION=ON;-DCMAKE_CXX_FLAGS_RELEASE=/MD /O2 /Oi /Gy /DNDEBUG /Zi ;-DCMAKE_C_FLAGS_RELEASE=/MD /O2 /Oi /Gy
>  /DNDEBUG /Zi ;-DCMAKE_SHARED_LINKER_FLAGS_RELEASE=/DEBUG /INCREMENTAL:NO /OPT:REF /OPT:ICF ;-DCMAKE_EXE_LINKER_FLAGS_RE
> LEASE=/DEBUG /INCREMENTAL:NO /OPT:REF /OPT:ICF ;-G;Visual Studio 14 2015 Win64;-DCMAKE_BUILD_TYPE=Release;-DCMAKE_INSTAL
> L_PREFIX=L:/vcpkg/packages/ceres_x64-windows
>     Working Directory: L:/vcpkg/buildtrees/ceres/x64-windows-rel
>     See logs for more information:
>       L:\vcpkg\buildtrees\ceres\config-x64-windows-rel-out.log
>       L:\vcpkg\buildtrees\ceres\config-x64-windows-rel-err.log
> 
> Call Stack (most recent call first):
>   scripts/cmake/vcpkg_configure_cmake.cmake:170 (vcpkg_execute_required_process)
>   ports/ceres/portfile.cmake:39 (vcpkg_configure_cmake)
>   scripts/ports.cmake:72 (include)
> 
> 
> Error: Building package ceres:x64-windows failed with: BUILD_FAILED
> Please ensure you're using the latest portfiles with `.\vcpkg update`, then
> submit an issue at https://github.com/Microsoft/vcpkg/issues including:
>   Package: ceres:x64-windows
>   Vcpkg version: 0.0.86-2017-09-04-97104258a782630a1a0d9a2504a533a550c50b69
> 
> Additionally, attach any relevant sections from the log files above.

There are writed that imported project "L:\Microsoft.Cpp.Default.props" not found:

> CMake Error at CMakeLists.txt: 40 (project):
>   Failed to run MSBuild command: 
> 	C:/Program Files (x86)/MSBuild/14.0/bin/MSBuild.exe
>   to get the value of VCTargetsPath:
>     Microsoft (R) Build Engine version 14.0.23107.0
>     (C) Microsoft Corporation (Microsoft Corporation). All rights reserved.
> 	
> 	The assembly started on 05/09/2017 20:52:32.
>     The "L:\vcpkg\buildtrees\ceres\x64-windows-rel\CMakeFiles\3.9.1\VCTargetsPath.vcxproj" project in node 1 (default targets).
>     L:\vcpkg\buildtrees\ceres\x64-windows-rel\CMakeFiles\3.9.1\VCTargetsPath.vcxproj (15,2): error MSB4019: the imported project "L:\Microsoft.Cpp.Default.props" was not detected. Check the path in the <Import> declaration and the presence of the file on the disk.
>     The build of the project "L:\vcpkg\buildtrees\ceres\x64-windows-rel\CMakeFiles\3.9.1\VCTargetsPath.vcxproj" is completed (the default target objects) with an error.
>     
>     Build error.
> 	
> 	"L:\vcpkg\buildtrees\ceres\x64-windows-rel\CMakeFiles\3.9.1\VCTargetsPath.vcxproj" (default target) (1) ->
>       L:\vcpkg\buildtrees\ceres\x64-windows-rel\CMakeFiles\3.9.1\VCTargetsPath.vcxproj (15,2): error MSB4019: the imported project "L:\Microsoft.Cpp.Default.props" was not detected. Check the path in the <Import> declaration and the presence of the file on the disk.
>     
>         Warnings: 0
>         Errors: 1
>     
>     Elapsed time: 00: 00: 00.01
>   Exit code: 1

--- a/ports/ceres/CONTROL
+++ b/ports/ceres/CONTROL
@@ -1,4 +1,4 @@
 Source: ceres
-Version: 1.13.0-1
+Version: 1.13.0-2
 Build-Depends:suitesparse, eigen3, clapack, gflags, glog
 Description: non-linear optimization package

--- a/ports/ceres/portfile.cmake
+++ b/ports/ceres/portfile.cmake
@@ -1,23 +1,8 @@
-# Common Ambient Variables:
-#   CURRENT_BUILDTREES_DIR    = ${VCPKG_ROOT_DIR}\buildtrees\${PORT}
-#   CURRENT_PACKAGES_DIR      = ${VCPKG_ROOT_DIR}\packages\${PORT}_${TARGET_TRIPLET}
-#   CURRENT_PORT DIR          = ${VCPKG_ROOT_DIR}\ports\${PORT}
-#   PORT                      = current port name (zlib, etc)
-#   TARGET_TRIPLET            = current triplet (x86-windows, x64-windows-static, etc)
-#   VCPKG_CRT_LINKAGE         = C runtime linkage type (static, dynamic)
-#   VCPKG_LIBRARY_LINKAGE     = target library linkage type (static, dynamic)
-#   VCPKG_ROOT_DIR            = <C:\path\to\current\vcpkg>
-#   VCPKG_TARGET_ARCHITECTURE = target architecture (x64, x86, arm)
-#
-
 if(VCPKG_CRT_LINKAGE STREQUAL "static")
     message(FATAL_ERROR "Ceres does not currently support static CRT linkage")
 endif()
 
 include(vcpkg_common_functions)
-
-set(VCPKG_PLATFORM_TOOLSET "v140") # Force VS2015 because VS2017 compiler return internal error
-# eigen3\eigen\src\core\redux.h(237): fatal error C1001: An internal error has occurred in the compiler. [internal\ceres\ceres.vcxproj]
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH

